stablehlo/dialect/StablehloOps.td

```diff
@@ -484,15 +484,15 @@ def StableHLO_PopulationCountOp: StableHLO_UnaryElementwiseOp<"popcnt",
     [Pure, HLO_CompatibleOperandsAndResultType], HLO_IntTensor> {
   let summary = "PopulationCount operator";
   let description = [{
-    Returns the number of bits set in each operand element-wise.
+    Performs element-wise count of the number of bits set in the `operand`
+    tensor and produces a `result` tensor.
 
-    See
-    https://www.tensorflow.org/xla/operation_semantics#element-wise_unary_functions.
+    See:
+    https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md#stablehlopopcnt
 
     Example:
-
     ```mlir
-    %0 = stablehlo.popcnt %arg0 : tensor<2x2xi32>
+    %result = "stablehlo.popcnt"(%operand) : (tensor<4xi8>) -> tensor<4xi8>
```

Comment:
if pretty print is there you can choose that format, sorry for not pointing this out earlier, approving the pr though

Reply:
There is no pretty print available for this op currently. Thanks for the heads up though.